src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/components/Home.js

```diff
@@ -13,7 +11,7 @@ export class Home extends Component {
           <li><a href='https://facebook.github.io/react/'>React</a> for client-side code</li>
           <li><a href='http://getbootstrap.com/'>Bootstrap</a> for layout and styling</li>
         </ul>
-        <p>To help you get started, we've also set up:</p>
+        <p>To help you get started, we&apos;ve also set up:</p>
```

Comment:
This is a bit awkward. Maybe we should phrase it as `we have also set up` to avoid the weird apostrophe encoding, so it's easier to read in the source code.

Reply:
That's also fine.